apps/functions-subscription/src/domain/__tests__/trial.test.ts

```diff
@@ -23,6 +24,30 @@ describe('insertTrial', () => {
     expect(testEnv.trialWriter.insert).toBeCalledWith(aTrial);
   });
 
+  it('should return all trials', async () => {
```

Comment:
**suggestion**: the `it` description doesn't represent what you are testing here

Reply:
Updated description in eff99bbaf41355bc1a98ee131ab9a4d2694136f4